Source/WebCore/platform/network/cocoa/NetworkStorageSessionCocoa.mm

```diff
@@ -463,12 +463,7 @@ static void deleteAllHTTPCookies(CFHTTPCookieStorageRef cookieStorage)
 
     NSURL *cookieURL = url;
 
-    std::optional<Seconds> cookieCap;
-#if ENABLE(INTELLIGENT_TRACKING_PREVENTION)
```

Comment:
You'd leave for instance this but gate it behind an additional flag for non-Cocoa ports.